src/datajudge/db_access.py

```diff
@@ -632,21 +632,30 @@ def get_row_count(engine, ref, row_limit: int = None):
     return result, [selection]
 
 
-def _column_aggregate(engine, ref, column_operator):
+def _column(engine, ref, aggregate_operator: Callable = None):
```

Comment:
I'm not sure the name of this function reflects very well what's doing.

Also, you could properly annotate all the arguments.

Be mindful on how the docstring style is in the rest of the documentation, I don't remember having seen the style with the `@`'s you're using within the codebase.

Reply:
What do you think about `_get_column`? I initially wanted to follow the `_column_aggregate` scheme, but this doesn't translate very well, you're right.

Regarding the Docstring: Do we have a pre-configured style? This was the default in PyCharm.
This [post](https://stackoverflow.com/questions/3898572/what-are-the-most-common-python-docstring-formats) talks about different styles and the PEP 257 discussion. In my opinion, `reStructuredText` looks like the default in the community for me. What is your opinion?

Reply:
> What do you think about `_get_column`? I initially wanted to follow the `_column_aggregate` scheme, but this doesn't translate very well, you're right.

Yes, probably that conveys better the intention of the function.
> 
> Regarding the Docstring: Do we have a pre-configured style? This was the default in PyCharm. This [post](https://stackoverflow.com/questions/3898572/what-are-the-most-common-python-docstring-formats) talks about different styles and the PEP 257 discussion. In my opinion, `reStructuredText` looks like the default in the community for me. What is your opinion?

I believe we tend to use `numpy` version. Also, if you see most of the code, documentation is mostly absent on the constraints themselves and the documentation basically resides in the `add_xxx` methods of the requirements. I'll defer to @kklein for this.

Reply:
> Also, if you see most of the code, documentation is mostly absent on the constraints themselves and the documentation basically resides in the add_xxx methods of the requirements

Yes. The heuristic I've been trying to follow most recently was: 
- Enduser information goes to the `Requirement`'s `add_*` method - this often revolves around what the constraint is about and how to use it.
- Developer information goes into the `Constraint` definition - this often revolves around how the constraint _works_.